tests/test.html

```diff
@@ -75,19 +75,37 @@
 
         <p>
             <strong>clndr.constraints</strong> Test start and end constraints.
-            (4th of this month to the 12th of next month). Logs in the console.
+            (the 4th of this month to the 12th of next month). Logs in the console.
         </p>
         <div id="constraints" class="cal1"></div>
 
         <p>
+            <strong>clndr.prevNextMonthConstriants</strong> Test start and end constraints.
```

Comment:
@kylestetz I just pushed up those typo fixes. I'm going to test this branch once more but I think this is good to go and resolves a few bugs too.